docs/RS APP Platform/rs-app-shedule.md

```diff
@@ -1,4 +1,4 @@
-##  RS School App Shedule
```

Comment:
Seems that somebody edited these files manually using github. I just fixed it with `prettier`, because he doesn't like it.